.circleci/config.yml

```diff
@@ -258,7 +258,7 @@ commands:
             name: Set up python environment
             command: |
               pip3 install --upgrade pip
-              pip3 install wheel setuptools
+              pip3 install wheel setuptools pyyaml
```

Comment:
Why do we need this dep? Is it something we need to add to setup.py / requirements.txt?

Reply:
I had noticed some CI failures when not using this - I think it may be necessary on account of one of the new imports, I will check.